StudyBuddy/Models/UserModel.cs

```diff
@@ -0,0 +1,28 @@
+using StudyBuddy.Abstractions;
+using StudyBuddy.ValueObjects;
+
+namespace StudyBuddy.Models;
+
+public class User : IUser
+{
+
+    public User(UserId id, string name, UserFlags flags, UserTraits traits)
+    {
+        Id = id;
+        Name = name;
+        Flags = flags;
+        Traits = traits;
+    }
+
+    public UserId Id { get; }
+
+    public string Name { get; set; }
+
+    public UserFlags Flags { get; set; }
+
+    public UserTraits Traits { get; set; }
+
+    public static string GenerateGravatarUrl(UserId userId) =>
```

Comment:
Maybe transfer to UserManager?

Reply:
Function has to do with user traits, not managing them.